src/stores/vendor.ts

```diff
@@ -178,7 +200,16 @@ export const vendorsStore = defineStore('vendors', {
         this.vendor = data.data
         return data.data
       } catch (error) {
-        console.log(error)
+        console.error(error)
+      }
+    },
+
+    async fetchVendorMe() {
+      try {
+        const data = await getVendorMe()
+        return data.data
+      } catch (error) {
+        throw error
```

Comment:
If you want to use the error in the calling function then you need to throw it again. I changed it here for you.